components/insight/SRC/org/openmicroscopy/shoola/util/ui/colourpicker/RGBModel.java

```diff
@@ -76,50 +77,71 @@
 	/** The lookup table */
 	private String lut;
 	
+	/** The reverse intensity flag */
+	private boolean revInt;
+	
+	/** The original reverse intensity flag */
+	private boolean originalRevInt;
 	
     /**
      * Sets the value of the model to v without firing an event.
      * 
      * @param v value.
      */
-    private void setV(float v)
+    private void setV(float v, boolean reset)
```

Comment:
:+1: Maybe our developer docs should say to check "validate tag arguments" and consider members as visible if they are whatever the level *above* private is (don't think we include private in docgen).